src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -123,6 +123,19 @@ public function testDontValidateIfParentWithoutCascadeValidation()
         $this->assertNoViolation();
     }
 
+    public function testNotExistingConstraintIndex()
+    {
+        $object = $this->getMock('\stdClass');
```

Comment:
Can you see any advantage of creating a dummy double for stdClass rather than simply creating the object?